### ISSUE TITLE
Add a button to easily go into replay mode

### ISSUE THREAD

**Issue by eastham** [eastham/tar1090#2]
...assuming, that is, it's up to date (adsbexchange.com lags by 30 minutes)

**Comment by harpermann** [comment]
I notice adsbexchange has a replay button at the bottom of the buttons on the right side of the display.  They seem to have a PI image we can use: https://www.adsbexchange.com/ways-to-join-the-exchange/build-your-own/.  I'm going to try it for grins.

**Comment by eastham** [comment]
Cool, that button is new.  The raspi box has been using the adsbexchange
image for the past few years, with manual mods.
But, sounds like the 30 minute delay is there?

**Comment by harpermann** [comment]
Yes, if you hit the replay button you get the replay on main page with all aircraft.  That updates it's end time on 30 minute intervals.

**Comment by eastham** [comment]
Hmm, well, do you know of any replay facilities at other ADS-B
aggregators?  I'm only familiar with some of the other open-data ones and I
think they lock down historical data pretty hard.

**Comment by harpermann** [comment]
<img width="1404" alt="image" src="https://github.com/eastham/tar1090/assets/2547812/d07465ba-2693-48aa-9a32-057757996468">

This is from FlightRadar24 on a paid gold account.  Not sure what's required to upload to them.  

The image from adsbexchange installs fine on a Raspi 4 but doesn't have replay enabled so I'm not sure if the UI has it or not.  

I need to knock off for today and catch up with packing for our trip.

**Comment by harpermann** [comment]
Oh boy, not to confuse things, but I found FlightRadar24 has an App.  https://www.flightradar24.com/blog/playback-is-now-available-in-the-flightradar24-app/   If that keeps history more like we want and it's saved in files..  Might be worth a look.   Insurmountable opportunities!

**Comment by eastham** [comment]
No worries, go pack, none of this is urgent.  FR24 looks interesting, and
it seems that adsbx image has a way to feed it:
https://www.adsbexchange.com/adsbexchange-custom-image-other-feed-clients/

**Comment by eastham** [comment]
Looks like FR24 lags 5 minutes on the web client, that's probably ok...

**Comment by eastham** [comment]
flightaware seems to have no lag, but you need to go into a per-aircraft
view.  Also probably acceptable, and it seems that you can feed flightaware
from the  adsbx image too.

**Comment by eastham** [comment]
Finally, and I think Harper was hinting at this earlier, we could use a
single-aircraft query like this: https://globe.adsbexchange.com/?reg=n624ux,
which seems to be up-to-date.  Just need to make a quick UI for it, which
could be as easy as a link from appsheet.
